tests/test_makeTransmissionCurves.py

```diff
@@ -0,0 +1,122 @@
+#
+# LSST Data Management System
+#
+# Copyright 2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+import unittest
+import numpy as np
+
+import lsst.utils.tests
+from lsst.afw.geom import Point2D
+from lsst.obs.hsc import makeTransmissionCurves
```

Comment:
I don't see any changes to `python/lsst/obs/hsc/__init__.py` in this changeset. Yet this test must have passed?

Reply:
If it's not in `__init__.py`, you can still do the above style of import (or `import lsst.obs.hsc.makeTransmissionCurves`).  You could not do, say,
```
import lsst.obs.hsc as obsHsc
d = obsHsc.makeTransmissionCurves.getOpticsTransmission()
```
Given that this module isn't needed by the vast majority of the code that uses obs_subaru, I figured I'd leave it out of the default imports.  Happy to change that if you think users will find it surprising; I don't think it's a big deal either way.

Reply:
Of course --- you're importing the module itself. Duh, silly me.